catalog/app/utils/AWS/Signer.js

```diff
@@ -2,7 +2,7 @@ import SignerV4 from 'aws-sdk/lib/signers/v4'
 import * as React from 'react'
 import * as redux from 'react-redux'
 
-import * as Auth from 'containers/Auth'
+import * as authSelectors from 'containers/Auth/selectors'
```

Comment:
what's the reason for this change?

Reply:
Cycle dependency:
Layout -> NavBar -> CatalogSettings -> AWS -> Signer,Credentials etc -> Auth -> SignIn, SIgnOut etc -> Layout

Reply:
oh i see, thanks for catching this. but how was it working before?

Reply:
ES6 allows cycle dependencies. So, it's a working code, but it's dangerous and has lint warning